platforms/ide/ide-native/src/main/java/org/gradle/ide/visualstudio/internal/VisualStudioTargetBinary.java

```diff
@@ -32,115 +30,96 @@ public interface VisualStudioTargetBinary {
     /**
      * Returns the Gradle project path for this binary
      */
-    @Input
```

Comment:
This type is part of the mutable model and is no longer used (or should be used) as a @Nested bean.